finesse/gui/hardware_set/hardware_sets_view.py

```diff
@@ -75,6 +75,7 @@ def __init__(self) -> None:
         self._connected_devices: set[OpenDeviceArgs] = set()
         pub.subscribe(self._on_device_opened, "device.opening")
         pub.subscribe(self._on_device_closed, "device.closed")
+        pub.subscribe(self._on_device_error, "device.error")
```

Comment:
The original version used `f"device.error.{instance!s}"` as topic, and not just `f"device.error"`. Is this change on purpose?

Reply:
Yep. This handler will catch errors for *all* devices, as opposed to just the one specific type. `instance` is also passed as an argument, as well as part of the topic name, so we can distinguish devices that way.